app/src/main/java/com/topjohnwu/magisk/ktx/XAndroid.kt

```diff
@@ -63,7 +60,11 @@ val PackageInfo.processes
             receivers?.processNames.orEmpty() +
             providers?.processNames.orEmpty()
 
-val Array<out ComponentInfo>.processNames get() = mapNotNull { it.processName }
+val Array<out ComponentInfo>.processNames
```

Comment:
I'll keep this out for now as toggle isolated processes does not do anything yet.
Would be better to add this after some functionality is introduced.